tracee-ebpf/tracee/co-re/vmlinux.h

```diff
@@ -503,6 +503,24 @@ struct sockaddr_in6 {
 	__u32 sin6_scope_id;
 };
 
+struct msghdr {
```

Comment:
Now, this file, all tabs are expanded in the beginning (to real tabs). I think we should change that to tabs being expanded to spaces but, before that, would you mind complying with the rest of the file and starting your structs indentation with real tabs ? (just so all the file is the same for now).